src/app/app.module.ts

```diff
@@ -1,5 +1,10 @@
+
 import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { NgModule } from '@angular/core';
+import { FlexLayoutModule } from '@angular/flex-layout';
+
+import { MatToolbarModule } from '@angular/material/toolbar';
```

Comment:
Consider ordering this by external reference first, then internal, alphabetically :)